scripts/run_cu_md_schnet.py

```diff
@@ -25,7 +25,7 @@
     }
 
     dataset = {
-        "src": "data/data/2020_06_03_rattle_emt",
+        "src": "data/data/",
```

Comment:
Hmm, why are we renaming this?

Reply:
Oops must of deleted this while testing it on my end. Speaking of which all these are datasets not available to someone who wants to test it. We should do something like the `ase_example` where we generate and use that data as an example. I can update these if you agree.

Reply:
Yeah, makes sense. I have a bit of an inclination to keep EMT / relaxations out of this runner script. It might be cleaner to package a sample trajectory that the `download_data.sh` script downloads and can be used for this runner (like `train_example`)?

Either way, that's probably best for a separate change / pull request. Let's revert this here and focus primarily on the evaluation in this PR :)